src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -161,6 +162,14 @@ relies on script task and is a nice introduction to workflows.
 Scripts can also be used to decorate tasks with specific actions, we support
 <<_pre_post_clean,pre>>, <<_pre_post_clean,post>>, <<_pre_post_clean,clean>> and <<_selection,selection>> scripts.
 
+==== Python Script Engine
+
+Since there are many different versions of Python (mainly Python2 and Python3) which are not compatible, ProActive supports all the versions of python.
```

Comment:
I think it is more clear to say : 
"ProActive supports all the Python versions (Python2, Python3,etc)"

Reply:
Done!